test/functional/package/access.spec.js

```diff
@@ -24,15 +21,18 @@ module.exports = function() {
      * @param ok {boolean}
      */
     function checkAccess(auth, pkg, ok) {
-      it((ok ? 'allows' : 'forbids') +' access ' + auth + ' to ' + pkg, function() {
-        server.authstr = auth ? buildToken(auth) : undefined;
-        let req = server.getPackage(pkg);
-        if (ok) {
-          return req.status(404).body_error(/no such package available/);
-        } else {
-          return req.status(403).body_error(/not allowed to access package/);
+      test(
+        (ok ? 'allows' : 'forbids') +' access ' + auth + ' to ' + pkg,
```

Comment:
template literal can be used here.

Reply:
I was limiting this migration only to the test runner :) .. If I have to refactor all unit test I won't finish this year. But, I'll put in my notes.